### ISSUE TITLE
Can't tombstone out fact in a keyed set.

### ISSUE THREAD

**Issue by FrontierOtr** [ambiata/ivory#648]
Tombstone values can't specify keys, so there's no way to know which value set to collide with and overrule.
Either a keyed tombstone is required for the use in keyed sets, or a special flag which can be set inside a ThriftFactStruct to say, "Actually, I'm not a real value, but rather a tombstone".

**Comment by markhibberd** [comment]
Agreed. There needs to be a keyed tombstone. I thought there was already an issue for this, but I can't see it now.
